docs/guides/mfa/05_strict_lax.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: strict_lax_mfa
+title: Strict and Lax Multi-Factor Authentication
+sidebar_title: Enforce Multi-Factor Authentication
```

Comment:
```suggestion
id: enforce_mfa
title: Enforce Multi-Factor Authentication
```

I like your title better :) So let's make it simpler and have a single title and also change the document ID. This category is auto-generated with my "special approach" so the doc `id` change doesn't have a wider impact (this document hasn't been referenced anywhere either).